.drone.star

```diff
@@ -1620,7 +1620,7 @@ def genericCachePurge(name, cache_key):
   return {
     'kind': 'pipeline',
     'type': 'docker',
-    'name': 'purge_%s_build_artifact_cache' %(name),
+    'name': 'purge_%s' %(name),
```

Comment:
Long pipeline names like "purge_ocis-binary-amd64_build_artifact_cache_build_artifact_cache" seems to lead to a silent fail of drone pipeline....

Reply:
yes, there is an "unfortunate" 50-character limit!